web/beacon-app/package.json

```diff
@@ -49,7 +49,6 @@
     "@radix-ui/react-avatar": "^1.0.1",
     "@radix-ui/react-tooltip": "^1.0.3",
     "@rotational/beacon-core": "^2.0.1",
-    "@rotational/beacon-foundation": "latest",
```

Comment:
I noticed this package was removed, so I updated the package.json and ran a new yarn lock file.

Reply:
Thank you for catching this!